Src/MakingSense.SafeBrowsing/GoogleSafeBrowsing/GoogleSafeBrowsingUpdater.cs

```diff
@@ -89,60 +89,23 @@ public GoogleSafeBrowsingUpdater(GoogleSafeBrowsingConfiguration configuration)
                 _database.ExitBackOffMode();
             }
 
-            _database.MinimumWaitDuration = TimeSpan.FromSeconds(double.Parse((response.MinimumWaitDuration as string).TrimEnd('s')));
-            _database.Updated = DateTimeOffset.Now;
+            var minWaitDuration = TimeSpan.FromSeconds(double.Parse((response.MinimumWaitDuration as string).TrimEnd('s')));
+            var listUpdates = response.ListUpdateResponses.Select(x=> MapListUpdate(x));
```

Comment:
If you like, you can rewrite it as `var listUpdates = response.ListUpdateResponses.Select(MapListUpdate);`